### PR TITLE
[Done]Up&Going/Ch1/##Functions

### PR DESCRIPTION
**Pull request by mballeng91**
**Yes, I promise I've read the [Contributions Guidelines](https://github.com/getify/You-Dont-Know-JS/blob/master/CONTRIBUTING.md)** (please feel free to remove this line).

### CODE REVIEW

File: up & going/ch1.md [hunk 1]
```diff
@@ -564,13 +564,13 @@ El ciclo `for` tiene tres cláusulas: La cláusula de inicialización (`var i=0`
 
 Hay otras formas especializadas de ciclos cuya intención es iterar sobre valores específicos, tales como las propiedades de un objeto (ver capítulo 2) dónde la evaluación del condicional se da cuando todas las propiedades han sido procesadas. El concepto de "ciclo hasta que una condición falle" permanece sin importar cual es la forma del ciclo.
 
-## Functions
+## Funciones
 
-The phone store employee probably doesn't carry around a calculator to figure out the taxes and final purchase amount. That's a task she needs to define once and reuse over and over again. Odds are, the company has a checkout register (computer, tablet, etc.) with those "functions" built in.
+El empleado de la tiendo probablemente no cargue con sigo una calculadora para encontrar el total de impuestos y el total de la compra. Es una tarea que necesita definir una vez y repetirla durante muchas veces. Hay chances de que la compañía cuente con una máquina registradora (computador, table, etc.) con estas "funciones" integradas.
```
Comment (Zenfeuer) [inline up & going/ch1.md]:
```suggestion
El empleado de la tienda probablemente no cargue consigo una calculadora para encontrar el total de impuestos y el total de la compra. Es una tarea que necesita definir una vez y repetirla durante muchas veces. Hay chances de que la compañía cuente con una máquina registradora (computador, tablet, etc.) con estas "funciones" integradas.
```

File: up & going/ch1.md [hunk 2]
```diff
@@ -564,13 +564,13 @@ El ciclo `for` tiene tres cláusulas: La cláusula de inicialización (`var i=0`
...
 
-Similarly, your program will almost certainly want to break up the code's tasks into reusable pieces, instead of repeatedly repeating yourself repetitiously (pun intended!). The way to do this is to define a `function`.
+De manera similar, su programa seguramente divididirá las tareas del código en piezas reutilizables, envés de repetidamente repetirse a si mismo repetitivamente (¡juego de palabras!).
```
Comment (Zenfeuer) [inline up & going/ch1.md]:
```suggestion
De manera similar, su programa seguramente dividirá las tareas del código en piezas reutilizables, envés de repetidamente repetirse a si mismo repetitivamente (¡juego de palabras!). La manera de hacer esto es definiendo una función.
```

File: up & going/ch1.md [hunk 3]
```diff
@@ -629,7 +629,7 @@ amount = calculateFinalPurchaseAmount( amount );
 console.log( amount.toFixed( 2 ) );		// "107.99"
 ```
 
-Although `calculateFinalPurchaseAmount(..)` is only called once, organizing its behavior into a separate named function makes the code that uses its logic (the `amount = calculateFinal...` statement) cleaner. If the function had more statements in it, the benefits would be even more pronounced.
+A pesar de que `calculateFinalPurchaseAmount(..)` es llamada una única vez, organizar esta rutina dentro de una función separada y con nombre hace el código que use su lógica (la sentencia `amount = calculateFinal...`) más limpia. Si la función tiene más sentencias dentro de ella, los beneficios son aún más pronunciados.
```
Comment (Zenfeuer) [inline up & going/ch1.md]:
```suggestion
A pesar de que `calculateFinalPurchaseAmount(..)` es llamada una única vez, organizar esta rutina dentro de una función separada y con nombre hace el código que use su lógica (la sentencia `amount = calculateFinal...`) sea más limpio. Si la función tiene más sentencias dentro de ella, los beneficios son aún más pronunciados.
```

**Comment by mballeng91** [comment]
@Zenfeuer : Muchas gracias por la revisión!  👍

**Pull request merged**